<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>showRoomsOwnerVersion</class>
 <widget class="QDialog" name="showRoomsOwnerVersion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1108</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>610</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>1031</width>
     <height>451</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>number of beds</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>description</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>status</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>edit</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>delete</string>
    </property>
   </column>
  </widget>
  <widget class="QComboBox" name="status_filter">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>50</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="number_of_beds_filter">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>50</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>20</y>
     <width>111</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Status filter</string>
   </property>
  </widget>
  <widget class="QComboBox" name="name_filter">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>111</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Room name filter</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>20</y>
     <width>141</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of beds filter</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>30</y>
     <width>82</width>
     <height>65</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="submit_button">
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="reset_button">
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
